tests/benchdnn/matmul/matmul.cpp

```diff
@@ -672,6 +672,30 @@ void skip_unimplemented_prb(const prb_t *prb, res_t *res) {
             return;
         }
     }
+    for (int arg : {DNNL_ARG_SRC, DNNL_ARG_WEIGHTS, DNNL_ARG_DST}) {
```

Comment:
I'd probably do it this way:
```
    if (amd || nvidia) {
        if (prb->attr.scales.has_host_scales()) {
            skipped;
            return ;
        }
    }
```

Reply:
Just to clarity, currently there is no such function in `zero_points_t` or `arg_scales_t` structures in benchdnn.
Is the suggestion to introduce one to hide `for (int arg : {DNNL_ARG_SRC, DNNL_ARG_WEIGHTS, DNNL_ARG_DST})` and policy check inside this function?